src/module/components/ngx-auth-firebaseui/auth.component.html

```diff
@@ -130,11 +123,15 @@
               <div fxLayout="column">
                 <mat-form-field [appearance]="appearance">
                   <mat-label>{{passwordText}}</mat-label>
-                  <input matInput
-                         [type]="toggle.type"
-                         name="password"
-                         [formControl]="sigUpPasswordFormControl"
-                         required>
+                  <input
+                    matInput
+                    [type]="toggle.type"
+                    name="password"
+                    [formControl]="sigUpPasswordFormControl"
+                    required
+                    [minlength]="min"
+                    [maxlength]="max"
```

Comment:
I think since we are using reactive form and we are indicating the min and max length via the form controller.. we can leave this part out

```html

[minlength]="min"
[maxlength]="max"

`
``

Reply:
Nope, validators will valide the min and max length but not block user input when you have reached max length. You need both. At least for max length.

Reply:
YES! Thanks for pointing that! awesome 🍻